<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng Kí</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #D3D3D3; /* Màu nền xám */
}

QLabel {
    color: #32CD32; /* Màu chữ xanh tươi sáng */
}

QPlainTextEdit {
    background-color: #F5F5F5; /* Màu nền cho PlainTextEdit */
    border: 1px solid #32CD32; /* Viền xanh tươi sáng */
    color: #000000; /* Màu chữ đen */
}

QPushButton {
    background-color: #32CD32; /* Nền xanh tươi sáng */
    color: white; /* Màu chữ trắng */
    border-radius: 5px; /* Bo tròn góc */
    padding: 10px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #228B22; /* Màu xanh đậm khi hover */
}

QLineEdit {
    background-color: #F5F5F5; /* Màu nền cho LineEdit */
    border: 1px solid #32CD32; /* Viền xanh tươi sáng */
    color: #000000; /* Màu chữ đen */
    padding: 5px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>301</width>
      <height>184</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Đăng Kí</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tên Đăng Nhập</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mật Khẩu</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nhập Lại Mật Khẩu</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Đăng Kí</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
